### PR TITLE
Set appropriate `endow_designation` fields' types to `EndowDesignation` 

### PR DESCRIPTION
**Pull request by misicnenad**
## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes

### CODE REVIEW

**Comment by SovereignAndrey** [comment]
@misicnenad Since the `endow_designation` field should hold a single value as a string and the form selection options are what's derived from the `EndowDesignation` type, I don't _think_ we need to make this change?

**Comment by misicnenad** [comment]
> @misicnenad Since the `endow_designation` field should hold a single NPO Type value as a string and it's the form field's selection options that are derived from the `EndowDesignation` type, I don't _think_ we need to make this change?

@SovereignAndrey we don't **need** to, but it will improve type-safety slightly (better safe than sorry).

File: src/constants/common.ts [hunk 1]
```diff
@@ -9,8 +12,12 @@ export const SEO_IMAGE =
 export const EMAIL_SUPPORT = "support@angelgiving.io";
 export const GENERIC_ERROR_MESSAGE = `An error occurred. Please get in touch with ${EMAIL_SUPPORT} if the problem persists.`;
 
-export const ENDOW_DESIGNATIONS = [
-  { label: "Charity", value: "Non-Profit", icon: "Charity" },
```
Comment (misicnenad) [inline src/constants/common.ts:13]: @SovereignAndrey do these "Non/For-Profits" mentions also need to be renamed?
https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/App/Header/DesktopNav.tsx#L10
https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/App/Footer/constants.ts#L67
https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/App/Header/MobileNav/Menu/Links.tsx#L24
https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/components/donation/Steps/Result/Success/Share.tsx#L65
https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/pages/Marketplace/Sidebar/Types.tsx#L7

Comment (SovereignAndrey) [reply src/constants/common.ts:13]: > @SovereignAndrey do these "Non/For-Profits" mentions also need to be renamed?
> 
> * https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/App/Header/DesktopNav.tsx#L10
> * https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/App/Footer/constants.ts#L67
> * https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/App/Header/MobileNav/Menu/Links.tsx#L24
> * https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/components/donation/Steps/Result/Success/Share.tsx#L65
> * https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/pages/Marketplace/Sidebar/Types.tsx#L7

No, those can all stay as is, as there we're speaking about all Non-profits on the platform (in general terms).

**Review by SovereignAndrey — APPROVED** [review]

**Review by ap-justin — APPROVED** [review]

**Pull request merged**